### ISSUE TITLE
`latest-release` is not updated since version `1.1.360`

### ISSUE THREAD

**Issue by tylerlaprade** [microsoft/pylance-release#6122]
https://github.com/microsoft/pylance-release/blob/main/releases/latest-release.json

This should be `1.1.371`.

See also: https://github.com/jakebailey/pyright-action/issues/133

**Comment by debonte** [comment]
Thanks for pointing this out @tylerlaprade! We've recently changed how our release automation works, so I assume this is a bug in those changes. 

In the meantime, I just manually updated the file.

**Comment by debonte** [comment]
Looks like we're always updating `latest-prerelease.json`, even if the release is a stable release. Here are links to recent release logs:

- 2024.7.1: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=9844450&view=logs&j=43d22c99-09f8-525c-20b6-0529e316eac7&t=ba33ce4f-a826-58d3-50f8-d1fa825fc934
- 2024.6.104: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=9830182&view=logs&j=43d22c99-09f8-525c-20b6-0529e316eac7&t=ba33ce4f-a826-58d3-50f8-d1fa825fc934
- 2024.6.1: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=9688648&view=logs&j=9b428885-a2ef-5956-4f5c-9c81452bd43a&t=11519467-3c90-566d-22fb-da4e7e7b375a

Note that in both the stable releases (2024.6.1, 2024.7.1) and the prerelease (2024.6.104), we're modifying `releases/latest-prerelease.json`:

```text
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	new file:   releases/2024.7.1.json
	modified:   releases/latest-prerelease.json
```

**Comment by debonte** [comment]
Ah, the problem is that AzDO is passing `False` (capitalized) for the `isPrerelease` option on `savePyrightVersionMetadata.js` and apparently `yargs` isn't handling that the way we expected:

- `--isPrerelease false`: Updates `latest-release.json`
- `--isPrerelease False`: Updates `latest-prerelease.json`

**Comment by debonte** [comment]
I believe I've fixed this, but I'll leave this open until our next stable release ships and verify at that point.

**Comment by debonte** [comment]
Closing. Confirmed that the fix worked as expected when 2024.8.1 shipped -- https://github.com/microsoft/pylance-release/commit/59c07ab6322b89b8d35b991526597286e971e0c8

Thanks for reporting this issue @tylerlaprade!
